pages/Profile/index.tsx

```diff
@@ -1,13 +1,42 @@
-import React from "react";
+import React from 'react';
 import Layout from "../../components/Layout/Layout";
-import styles from "./Profile.module.css";
+import styles from "./Profile.module.css" ;
+import Button from "@mui/material/Button";
+import { AboutInfo } from '../../components/ProfileInfo/AboutInfo';
+import { LoginDetails } from '../../components/ProfileInfo/LoginDetails';
+import { Typography } from '@mui/material';
+import router from 'next/router';
 
 const ProfileSettingsPage: React.FunctionComponent = () => {
-  return (
-    <Layout title="Profile">
-      <main className={styles.main}>Profile Settings</main>
-    </Layout>
-  );
-};
+    
+    const displayinfo = () => {
+        return (
+            <div className={styles.profile}>
+                <div className = {styles.namebar}>
+                    <h1> Cindy Zhang </h1>
+                    <Button className={styles.button} onClick={() => {router.push(`./EditProfile`);}}>Edit</Button>
```

Comment:
Since we will be deleting the edit profile page, just make sure this button does not route to /EditProfile